### PR TITLE
Add MPTTester class to streamline MPToken unit-tests

### PR DESCRIPTION
**Pull request by gregtatcam**

### CODE REVIEW

File: src/test/jtx/mpt.h [hunk 1]
```diff
@@ -20,46 +20,177 @@
...
+};
+
+class MPTTester
+{
+    Env& env_;
+    Account const& issuer_;
+    std::unordered_map<std::string, AccountP> const holders_;
+    std::optional<std::uint32_t> sequence_;
+    std::optional<uint192> id_;
+    std::optional<uint256> issuanceID_;
```
Comment (shawnxie999) [inline src/test/jtx/mpt.h]: i think this should be called something like `issuanceKey_`

Comment (gregtatcam) [reply src/test/jtx/mpt.h]: And `id_` should be `issuanceID_`, right?

Comment (shawnxie999) [reply src/test/jtx/mpt.h]: i dont think it matters...

**Review by shawnxie999 — APPROVED** [review]
Thanks for the refactor!

**Pull request merged**